pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 
```

Comment:
it feels a bit clunky that the above image is between "the following statuses" and the table of statuses

Reply:
agreed - I also feel the image needs a description as to what it's showing? it took me a while to connect the colourful bar as the statuses.

Reply:
Agreed, I think it's actually clearer if I remove that "fleet" summary image, it doesn't add anything and is fairly unrelated to device status